### PR TITLE
Issue#181 Develop the nav bar

### LINKED ISSUE #181: Develop the Navigation Bar
## Description
As a contributor,
I need develop the navigation bar,
so that I can navigate easily in the website.

### Mockup
navbar

### Update [Required]
Addition of a new file for nav-bar.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5 hours

@Janiceilene @divyanshu-rawat @ancyp Please assign me the issue to work upon! :)

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Adding the nav-bar & implemented routing in Angular 5.

Fixes #181 & #118 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface  

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I ran it on my local computer.
Mockup Attached : 
screen shot 2018-05-28 at 8 58 44 am

screen shot 2018-05-28 at 8 58 58 am

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 
  

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules
 
It should be merged after #183

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@BaaniLeen I guess adding footer at app.component.html will also work well then we can have router-outlet in between navbar and footer. 
what do you think?

File: e2e/protractor.conf.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  capabilities: {
+    "browserName": "chrome"
+  },
+  directConnect: true,
+  baseUrl: "http://localhost:4200/",
+  framework: "jasmine",
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000,
+    print: function() {}
```
[the remark this answers is not in the document]
Comment (ancyp) [reply e2e/protractor.conf.js]: https://eslint.org/docs/rules/object-shorthand @BaaniLeen

**Comment by BaaniLeen** [comment]
@Tharangi I tried minimising the icons but they become way too small then. Is the current size workable?
Also, I will update the active page highlight thing :)

**Comment by BaaniLeen** [comment]
@ancyp The conflicts are because the initial file had a hello world template in the main Html file, while in the new one with the nav bar, we have introduced components and done routing :)
I will resolve them :)

**Comment by ancyp** [comment]
screen shot 2018-05-26 at 7 53 35 pm
Please attach both mockup and actual screenshot in the desc.

File: src/app/navbar/navbar.component.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <li class="nav-item waves-light" mdbRippleRadius>
+        <!--Medium-->
+        <a type="button" class="nav-link btn-floating btn-lg btn-fb" target="_blank" href="https://medium.com/@systers_org"><i class="fa fa-medium fa-2x"></i></a>
+    </li>
+    </ul>
+    <!--End of Contact Us Social Media Icons-->
+    </links>
+    </mdb-navbar>
+
+    <script>
```
Comment (ancyp) [inline src/app/navbar/navbar.component.html]: this part does not seem to work.
Also it is not good practice to include script in html file.

**Comment by BaaniLeen** [comment]
@ancyp Okay sure. I forgot to update the description after changing the colour scheme. :)

**Comment by BaaniLeen** [comment]
@ancyp @Tharangi @divyanshu-rawat All the suggestions have been incorporated & the merge conflicts resolved. Can you please merge the PR now? :)

**Pull request merged**